Examples/Tests/ionization/PICMI_inputs_2d.py

```diff
@@ -47,12 +47,14 @@
     fill_in = True)
 electrons = picmi.Species(
     particle_type = 'electron',
-    name = 'electrons')
+    name = 'electrons',
+    warpx_add_real_attributes = {'orig_z': 'z'})
 ions = picmi.Species(
     particle_type = 'N',
     name = 'ions',
     charge_state = 2,
-    initial_distribution = uniform_distribution)
+    initial_distribution = uniform_distribution,
+    warpx_add_real_attributes = {'orig_z': 'z'})
```

Comment:
This is needed otherwise the `ParticleDiagnostic` crashes due to 'ions' not having an `orig_z` attribute.